mergin/client.py

```diff
@@ -303,6 +344,12 @@ def projects_list(self, tags=None, user=None, flag=None, q=None):
         :param q: Search query string
         :type q: String
 
+        :param page: Page number for paginated projects list
+        :type page: Integer
+
+        :param per_page: Number of projects fetched per page, max 100 (restriction set by server)
+        :type per_page: Integer
+
```

Comment:
This comment is a leftover probably from a previous commit?

Maybe we should write there that this will always return max. N (100?) results, and that it is recommended to use paginated variant when more projects may be listed.

Reply:
Done.